[IMP] sales: updated pro-forma invoice doc (#4496)
odoo/documentation: content/applications/sales/sales/invoicing/proforma.rst

==================
Pro-forma invoices
==================

> A *Pro-Forma Invoice* is an abridged or estimated invoice sent in advance of a delivery of goods.
> It notes the kind and quantity of goods, their value, and other important information, such as
> weight and transportation charges.

Reviewer: Since this is not a proper noun and is not an Odoo exclusive feature, I don't believe it gets capitalized, as well as an early line break 'it' can be on line 5 :)
```suggestion
A *pro-forma invoice* is an abridged or estimated invoice sent in advance of a delivery of goods. It
notes the kind and quantity of goods, their value, and other important information, such as weight
and transportation charges.
```

Pro-forma invoices are commonly used as preliminary invoices with a quotation. They are also used
for customs purposes during importation. They differ from a normal invoice in that they are not a

> demand (or request) for payment.

Reviewer 2: This makes me wonder what the difference is between a pro-forma invoice and a quotation. It might be good to add a brief explanation.

Author: I understand, but I do think the explanation can be found on lines 5-7. And, considering this doc is solely focused on pro-forma invoices, I believe the reader/audience is already aware of what a common quotation is...but again, I completely get where you're coming from. But, in this instance, I think, for the sake of staying as concise as possible, I'm going to leave this portion as-is. But, I genuinely do appreciate the suggestion. Thanks! 👍

Activate the feature
====================

> In order to utilize pro-forma invoices, the :guilabel:`Pro-Forma Invoice` featutre **must** be

Reviewer 2: Typo
```suggestion
In order to utilize pro-forma invoices, the :guilabel:`Pro-Forma Invoice` feature **must** be
```

> In order to utilize pro-forma invoices, the :guilabel:`Pro-Forma Invoice` feature **must** be
> activated. To do that, navigate to :menuselection:`Sales app --> Configuration --> Settings`, and
> in the :guilabel:`Quotations & Orders` section, click the checkbox next to :guilabel:`Pro-Forma
> Invoice`. Then, click :guilabel:`Save` to save all changes.

Reviewer: using italics since the active instruction hasn't begun yet
```suggestion
In order to utilize pro-forma invoices, the *Pro-Forma Invoice* feature **must** be activated. To do
that, navigate to :menuselection:`Sales app --> Configuration --> Settings`, and in the
:guilabel:`Quotations & Orders` section, click the checkbox next to :guilabel:`Pro-Forma Invoice`.
Then, click :guilabel:`Save` to save all changes.
```

> Invoice` feature. Then, click :guilabel:`Save` to save all changes.

Reviewer 2: Remove word
```suggestion
Invoice`. Then, click :guilabel:`Save` to save all changes.
```

> .. image:: proforma/pro-forma-setting.png
>     :align: center
>     :alt: The pro-forma feature setting in the Odoo Sales application.

Reviewer: Image attribute alignment and using proper casing for the field name in the alt tag
```suggestion
.. image:: proforma/pro-forma-setting.png
   :align: center
   :alt: The Pro-Forma Invoice feature setting in the Odoo Sales application.
```

Send a pro-forma invoice
========================

> With the :guilabel:`Pro-Forma Invoice` feature activated, the option to send a pro-forma invoice is

Reviewer: I think this can also be italics, but may be arguable that a gui can be used too since it is part of the active instruction
```suggestion
With the *Pro-Forma Invoice* feature activated, the option to send a pro-forma invoice is
```

> now available on any quotation or sales order, via the :guilabel:`Send Pro-Forma Invoice` button.

Reviewer 3: Suggestion
```suggestion
now available on any quotation or sales order, via the :guilabel:`Send Pro-Forma Invoice` button, located at the top of a sales order form.
```

> .. image:: proforma/send-pro-forma-invoice-button.png
>     :align: center
>     :alt: The pro-forma invoice button on a typical sales order in Odoo Sales.

Reviewer: image attribute alignment and field name capitalization in alt tag again
```suggestion
.. image:: proforma/send-pro-forma-invoice-button.png
   :align: center
   :alt: The Send Pro-Forma Invoice button on a typical sales order in Odoo Sales.
```

When the :guilabel:`Send Pro-Forma Invoice` button is clicked, an email pop-up appears. The

> :guilabel:`Recipients` field is autopopulated with the customer from the sales order or quotation.

Reviewer 2: ```suggestion
:guilabel:`Recipients` field is auto-populated with the customer from the sales order or quotation.
```

Then, if needed, modify the :guilabel:`Subject` field and the body of the email. The pro-forma
invoice is automatically added as an attachment to the email.

> When ready, click send and Odoo automatically sends the email, with the attached pro-forma invoice,
> to the customer.

Reviewer 2: Add GUI label. Also, does it automatically send if you have to click Send to make it happen? It might be good to phrase that differently.
```suggestion
When ready, click :guilabel`Send` and Odoo automatically sends the email, with the attached pro-forma invoice,
to the customer.
```

> .. image:: proforma/pro-forma-email-message-popup.png
>     :align: center
>     :alt: The email pop-up window that appears with pro-forma invoice attached in Odoo Sales.

Reviewer: image attribute alignment, make sure to place the :align: and :alt: attributes under the 'i' of 'image' :)
```suggestion
.. image:: proforma/pro-forma-email-message-popup.png
   :align: center
   :alt: The email pop-up window that appears with pro-forma invoice attached in Odoo Sales.
```

.. tip::
   To preview what the pro-forma invoice looks like, click on the PDF at the bottom of the email
   pop-up window *before* clicking :guilabel:`Send`. When clicked, the pro-forma invoice is
   downloaded instantly. Open that PDF to view (and review) the pro-forma invoice.

>    

Reviewer: Remove spaces keep the line break (trailing spaces will cause build errors)  
```suggestion

```

Author: ah, darn it! haha I'll pay more attention to those pesky trailing spaces. Fixing now.

> .. tip::
>    To preview what the pro-forma invoice looks like, click on the pdf at the bottom of the email
>    pop-up window *before* clicking :guilabel:`Send`. When clicked, the pro-forma invoice is
>    downloaded instantly. Open that pdf to view (and review) the pro-forma invoice.
> 
>    .. image:: proforma/pro-forma-pdf.png
>       :align: center
>       :alt: Sample pro-forma invoice pdf from Odoo Sales.

Reviewer: Capitalizing 'PDF' since it is an acronym, this may be a stylistic choice but I wanted to state my thoughts :)
```suggestion
.. tip::
   To preview what the pro-forma invoice looks like, click on the PDF at the bottom of the email
   pop-up window *before* clicking :guilabel:`Send`. When clicked, the pro-forma invoice is
   downloaded instantly. Open that PDF to view (and review) the pro-forma invoice.

   .. image:: proforma/pro-forma-pdf.png
      :align: center
      :alt: Sample pro-forma invoice PDF from Odoo Sales.
```
